src/BuildScriptGenerator/Node/NodeScriptGeneratorOptions.cs

```diff
@@ -13,6 +13,12 @@ public class NodeScriptGeneratorOptions
         /// </summary>
         public string CustomRunBuildCommand { get; set; }
 
+        /// <summary>
+        /// Gets or sets custom build command or script that will run without 'npm install' in the generated
+        /// build script.
+        /// </summary>
```

Comment:
I'm bit confused .. what is the difference between a customrunbuildcommand and a custombuildcommand

Reply:
So the difference is basically custombuildcommand will override any PackageInstallCommand Oryx provided(either could be 'npm install' or 'yarn install')